xsnap/sources/xsnap-worker.c

```diff
@@ -72,20 +71,6 @@ xsCallback gxSnapshotCallbacks[mxSnapshotCallbackCount] = {
 	xs_performance_now, // 4
 	xs_currentMeterLimit, // 5
 	xs_resetMeter, // 6
-
-	xs_textdecoder, // 7
-	xs_textdecoder_decode, // 8
-	xs_textdecoder_get_encoding, // 9
-	xs_textdecoder_get_ignoreBOM, // 10
-	xs_textdecoder_get_fatal, // 11
-
-	xs_textencoder, // 12
-	xs_textencoder_encode, // 13
-	xs_textencoder_encodeInto, // 14
-
-	xs_base64_encode, // 15
-	xs_base64_decode, // 16
-
```

Comment:
I don't think @phoddie meant to remove TextEncoder / TextDecoder ; it looks like some sort of version skew set in.